### PR TITLE
chore: fix broken import after dac183a1

### PR DESCRIPTION
**Pull request by kleesc**

### CODE REVIEW

File: endpoints/api/repository.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 from flask import request, abort
 
-from app import dockerfile_build_queue, tuf_metadata_api, repository_gc_queue
```
[the remark this answers is not in the document]
Comment (dmage) [reply endpoints/api/repository.py:13]: I mean this PR won't change anything.

[the remark this answers is not in the document]
Comment (kleesc) [reply endpoints/api/repository.py:13]: Didn't these get moved into `singletons/` ow wherever they are now?

[the remark this answers is not in the document]
Comment (dmage) [reply endpoints/api/repository.py:13]: They are moved, but to keep all `app` consumers functional, they are also imported into `app`. If you import them via `singletons.*`, only needed packages will be imported, if you import them via `app`, all packages will be imported.

[the remark this answers is not in the document]
Comment (kleesc) [reply endpoints/api/repository.py:13]: ok

**Pull request closed without merging**